cfme/tests/infrastructure/test_providers.py

```diff
@@ -222,6 +223,35 @@ def test_provider_crud(provider):
     provider.wait_for_delete()
 
 
+@pytest.mark.usefixtures('has_no_infra_providers')
+@pytest.mark.tier(1)
+@test_requirements.provider_discovery
+@pytest.mark.parametrize('verify_tls', [False, True])
+@pytest.mark.uncollectif(lambda provider:
+    not provider.one_of(RHEVMProvider) and not provider.data['verify_tls'])
+def test_provider_rhv_create_delete_tls(provider, verify_tls):
+    """Tests RHV provider creation with and without TLS encryption
+
+    Metadata:
+       test_flag: crud
+    """
+    p = copy(provider)
+
+    if not verify_tls:
+        endpoints = deepcopy(p.endpoints)
+        endpoints['default'].verify_tls = False
```

Comment:
I understand that by default the ``endpoint.verify_tls`` is true, so testing with TLS is performed every time the provider is created (for other tests etc.). Wouldn't it be enough to test just provider creation without TLS in this test?